### DISCUSSION TITLE
Hw8 Q1, eom_sys

### DISCUSSION THREAD

**Discussion by RebeccaVillanueva** [cooperrc/advanced-dynamics#50] [Q&A]
Does anyone understand how to go about making a function for the final equation of motion? Maybe I just don't understand the embedding technique enough but I'm just not sure how to make a function that returns an equation.

**Comment by cooperrc** [comment]
For the embedding method, the goal is to give it time and state and get derivatives of state e.g. 

```python
>>> eom_sys(t, [x, theta, dxdt, dthetadt])
[dxdt, dthetadt, ddxddt, ddthetaddt]
```

the equations of motion are defined mostly by the $B$ matrix $[6\times2]$ and $Q_{e}$ force vector, length 6

**Reply by RebeccaVillanueva** [reply]
Thank you!
